### ISSUE TITLE
Getting below error when i enter the no specified and click on call button

### ISSUE THREAD

**Issue by Bond-Prabal** [DanWahlin/acs-voice-calling-quickstart#1]
re:ACS:warning 3/12/2021, 12:38:4:998 [CallClient] [JS.TsCalling.MediaAgent] ffffffff: MA/webrtc2-4 On ICE candidate error turn::443?transport=tcp | 192.168.0.x:65281: code 401: Unauthorized
CallClient.e.warning.log @ sdk.bundle.js:149
debug @ debug.js:122
warn @ sdk.bundle.js:149
t.log @ sdk.bundle.js:131
(anonymous) @ sdk.bundle.js:36
t.log @ sdk.bundle.js:36
e.warn @ sdk.bundle.js:36
(anonymous) @ sdk.bundle.js:131
e._apply @ sdk.bundle.js:131
e.warn @ sdk.bundle.js:131
peerConnectionPromise.relayManagerProvider.getRelayManager.queryRelaysAsync.then.u.onicecandidateerror @ sdk.bundle.js:131

**Comment by DanWahlin** [comment]
Looks like an unauthorized error. Did you try reissuing a new token and updating client.js with it? I've seen that error when the token was no longer valid.

**Comment by Bond-Prabal** [comment]
Yes of course!

**Comment by DanWahlin** [comment]
Glad that helped

**Comment by Bond-Prabal** [comment]
No. it did not work after use new token each time.

**Comment by DanWahlin** [comment]
I'd need more details then. I suspect something else is going on.

**Comment by DanWahlin** [comment]
Bummer - sorry that it didn't work for you (thought you were saying it did).

I just verified that everything is working as expected (re-cloned the repo and started from scratch). However, the first time I ran it I got a 401 as well. Second time (with a new token) the button was enabled and it worked fine. 3rd time (with a new token) I got a 401 again unfortunately. ACS is in preview right now and I'm wondering if the service is having an issue. 

I just ran it again and it all worked fine as far as the button being enabled (no 401 errors). But, the bot was very slow to respond. So, I'm still thinking there may be something going on with the service right now.

If you're sure you've gone through all of the steps in the readme correctly but are still getting the error even after adding a new token to client.js then you might try a little later (with a new token of course). Not a great answer, but I don't have any control over the service unfortunately (and it's still in "preview" as mentioned). I'm confident the code is working fine as long as the ACS connection string is added to the .env file and a valid token is used in client.js though.
